packages/lab/src/__tests__/__e2e__/dropdown-next/DropdownNext.cy.tsx

```diff
@@ -105,20 +105,20 @@ describe("GIVEN an active Dropdown component", () => {
 
   describe("WHEN the Dropdown is rendered with defaultSelected prop", () => {
     it("THEN it should show default selected value on first render", () => {
-      cy.mount(<Secondary />);
+      cy.mount(<WithDefaultSelected />);
 
-      cy.findByRole("combobox").should("have.value", "🇲🇬 Madagascar");
+      cy.findByRole("combobox").should("have.value", "California");
     });
 
     // TODO: update once KeyNav fixed in List
```

Comment:
If the todo is here, does it mean the test is testing wrong behaviour?

Reply:
the test right now matches what user would expect to see when they use dropdown - so yes cypress would need updating once ListNext is fixed to reflect the updated behaviour